### ISSUE TITLE
Module not found: Error: Can't resolve 'ejs-html-loader'

### ISSUE THREAD

**Issue by gulping** [akim-mcmath/ejs-html-loader#18]
after upgrade 3.1.0 → 4.0.0,

`Module not found: Error: Can't resolve 'ejs-html-loader' in 'repository path'`

----

`node_modules/ejs-html-loader`

on 3.1.0

```
lib/
.npmignore
LICENSE
package.json
package.lock.json
README.md
```

on 4.0.0

```
node_modules/
LICENSE
package.json
README.md
```

package.json `"main": "lib/index.js"` is missing on 4.0.0.

**Comment by mcmath** [comment]
That was a very silly error on my part. It's fixed now in version 4.0.1.

**Comment by gulping** [comment]
4.0.1 works well 👍
